src/main/java/io/appium/java_client/IOSMobileCommand.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I don't like this class. I think that it is better to just add the new command to MobileCommand.